lib/code.rb

```diff
@@ -1,15 +1,10 @@
-require_relative './commands'
-
 class Code
-  include Commands
-  attr_reader :red_pins, :white_pins
-  
-  CodeLength = 1..4
-  ColorBank = 6
+	CodeLength = 1..4
+ 	ColorBank = 6
   
-  def generate_code
-    @colors = colors = ["r", "g", "o", "y", "b", "p"]
-    @answer = (CodeLength).map { @colors[rand(ColorBank)] }
+	def generate_code
```

Comment:
not sure why the indent shows up like this here, it looks right in my text editor